dotnet/devops/Build.ps1

```diff
@@ -0,0 +1,64 @@
+param($Platform, $OutLocation, $AndroidNdkHome)
```

Comment:
Can prob leave this out as we have build.sh that does the same thing now?

Reply:
The build script is part of the azure pipelines flow, which currently builds and packages everything and allows us to publish the nuget to CI and nuget.org repos. I've opened an issue [here](https://github.com/mattrglobal/ffi-bbs-signatures/issues/13) to track migrating the build and ci scripts.
If OK, I'd like to keep this in until there's full functional CI.